egs/fisher_callhome_spanish/st1/local/normalize_trans.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
+# Copyright 2018 Kyoto University (Hirofumi Inaguma)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
```

Comment:
It's better to add summary descriptions of what kind of normalization this script performs.
Also, if possible, it would be great to have some examples.